bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/util/exceptions/BallerinaErrorReasons.java

```diff
@@ -74,6 +74,7 @@
     public static final String KEY_NOT_FOUND_ERROR_IDENTIFIER = "KeyNotFound";
     public static final String INVALID_TYPE_TO_SORT = "SortOperationError";
     public static final String UNORDERED_TYPES = "UnorderedTypesError";
+    public static final String LENGTH_SHOULD_BE_LESS_THAN_2147483647 = "length should be less than 2147483647";
```

Comment:
Shall we explain here why it is `2147483647`?
Maybe you can find a more informative error message

Reply:
Will update the error msg (https://github.com/ballerina-platform/ballerina-lang/pull/35598#discussion_r837624620).